### PR TITLE
Updates to RTD documentation based on full review

### LINKED ISSUE #1272: Review and update new RTD documentation
**Description**

The current contents of the global-workflow wiki have been transferred into the new Read-The-Docs documentation as-is and deployed into global-workflow `develop` @ 6713eaeb829df2cffbc0c959d17b1f14033d783c:

https://global-workflow.readthedocs.io/en/latest/index.html

Review the documentation, correct any errors, and update the content for recent system changes.

Once done, repurpose repo wiki.

Related to issue #9.
Closes #9

### PR DESCRIPTION
**Pull request by KateFriedman-NOAA**
**Description**

This PR updates the new RTD documentation that was moved from the global-workflow wiki. The wiki (and now RTD) needed to be reviewed for updates related to recent GFS/global-workflow system updates, as well as fix some spelling/syntax issues. This is the first documentation update since the new RTD documentation was added to the repo.

As part of the updates the following changes occurred:
- A few spelling and/or Sphinx syntax corrections.
- Introduce use of `note` and `warning` boxes.
- Review and update of content to match GFS system updates.
- Reorganize/rename some sections.

Preview of incoming updated documentation can be viewed here on RZDM:

https://www.emc.ncep.noaa.gov/gmb/emc.glopara/read-the-docs/global-workflow/issue1272/index.html

Note: Pages will look a bit different (e.g. missing bullets), likely due to no Sphinx installation on RZDM (?). Expect other similar subtle differences compared to [live RTD documentation page](https://global-workflow.readthedocs.io/en/latest/). I'm open to suggestions to make this preview look more like the live pages (e.g. install Sphinx?).

**Type of change**

Maintenance and regular updates to documentation.

**How Has This Been Tested?**

Viewed feature branch Git Action artifact generated html pages in browser.
  
Resolves #1272

### CODE REVIEW

File: docs/source/clone.rst [hunk 1]
```diff
@@ -32,31 +39,45 @@ Clone workflow and component repositories
...
 
-https method::
+https method:
+
+::
 
    git clone https://github.com/NOAA-EMC/global-workflow.git
 
-ssh method (using a password protected SSH key)::
+ssh method (using a password protected SSH key):
```
Comment (WalterKolczynski-NOAA) [inline docs/source/clone.rst:44]: Let's move this ahead of https and note it is the highly preferred method

File: docs/source/configure.rst [hunk 2]
```diff
@@ -2,70 +2,61 @@
...
 +----------------+------------------------------+---------------+-------------+---------------------------------------------------+
-| DO_AWIPS       | Run jobs to produce AWIPS    | NO            | YES         | downstream processing, ops only                   |
-|                | products                     |               |             |                                                   |
+| DO_GLDAS       | Run GLDAS to spin up land    | YES           | YES         | Spins up for 84hrs if sflux files not available   |
+|                | ICs                          |               |             |                                                   |
 +----------------+------------------------------+---------------+-------------+---------------------------------------------------+
-| WAFSF          | Run jobs to produce WAFS     | NO            | YES         | downstream processing, ops only                   |
-|                | products                     |               |             |                                                   |
+| DO_WAVE        | Runs wave jobs and produce   | YES           | YES         | One-way wave coupling, new for GFSv16             |
```
Comment (WalterKolczynski-NOAA) [inline docs/source/configure.rst]: This is control by APP and probably doesn't need to be on this list.

File: docs/source/monitor_rocoto.rst [hunk 3]
```diff
@@ -53,9 +70,10 @@ Usage
 
    ./rocoto_viewer.py -d /path/to/workflow/database/file -w /path/to/workflow/xml/file
 
-Note 1: Terminal/window must be wide enough to display all experiment information columns, viewer will complain if not.
+.. note::
+   Note 1: Terminal/window must be wide enough to display all experiment information columns, viewer will complain if not.
 
-Note 2: The viewer requires the full path to the database and xml files if you are not in your EXPDIR when you invoke it.
+   Note 2: The viewer requires the full path to the database and xml files if you are not in your EXPDIR when you invoke it.
```
Comment (WalterKolczynski-NOAA) [inline docs/source/monitor_rocoto.rst:76]: Note 3: Only `TERM=xterm` is supported. You may wish to create a shell function to switch automatically if you are in a different terminal:

Bash example
```
function rv {
	oldterm=${TERM};
	export TERM='xterm';
	${PATH_TO_VIEWER}/rocoto_viewer.py $@;
	export TERM=${oldterm};
}
```

**Review by WalterKolczynski-NOAA — CHANGES_REQUESTED** [review]

File: docs/source/configure.rst (hunk 2 above)
Comment (KateFriedman-NOAA) [reply docs/source/configure.rst]: Good point. Committed to branch at 3f9b7012.

File: docs/source/clone.rst (hunk 1 above)
Comment (KateFriedman-NOAA) [reply docs/source/clone.rst:44]: Done @ 75f19424

File: docs/source/monitor_rocoto.rst (hunk 3 above)
Comment (KateFriedman-NOAA) [reply docs/source/monitor_rocoto.rst:76]: Added @ 6be21829

**Comment by KateFriedman-NOAA** [comment]
Preview of RTD updated after addressing @WalterKolczynski-NOAA's comments: https://www.emc.ncep.noaa.gov/gmb/emc.glopara/read-the-docs/global-workflow/issue1272/index.html

**Review by WalterKolczynski-NOAA — APPROVED** [review]

**Pull request merged**